src/lib.rs

```diff
@@ -38,6 +38,16 @@
 //! [`Rc`]: https://doc.rust-lang.org/std/rc/struct.Rc.html
 //! [`Arc`]: https://doc.rust-lang.org/std/sync/struct.Arc.html
 //! [`PrivateKeyDer::clone_key()`]: https://docs.rs/rustls-pki-types/latest/rustls_pki_types/enum.PrivateKeyDer.html#method.clone_key
+//!
+//! ## Target `wasm32-unknown-unknown` with the `web` feature
+//!
+//! [`std::time::SystemTime`](https://doc.rust-lang.org/std/time/struct.SystemTime.html)
+//! is unavailable in `wasm32-unknown-unknown` targets, so calls to
+//! [`UnixTime::now()`](https://docs.rs/rustls-pki-types/latest/rustls_pki_types/struct.UnixTime.html#method.now),
+//! otherwise enabled by the [`std`](https://docs.rs/crate/rustls-pki-types/latest/features#std) feature,
+//! require building instead with the [`web`](https://docs.rs/crate/rustls-pki-types/latest/features#web)
+//! feature. It gets time by calling [`Date.now()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now).
```

Comment:
small nit: looks like there's an extra `.` here rendering as "It gets time by calling Date.now()**.** in the browser."


```suggestion
//! feature. It gets time by calling [`Date.now()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now)
```